webportal/src/app/create_item/create_item.jsx

```diff
@@ -27,21 +25,86 @@ const GrayCard = styled.div`
   box-shadow: rgba(0, 0, 0, 0.06) 0px 2px 4px, rgba(0, 0, 0, 0.05) 0px 0.5px 1px;
 `;
 
+const itemDescription = {
```

Comment:
Currently the UI does not show this section belongs to description clearly, I prefer to hide all those fields before it is more clear